src/foam/core/reflow/SinkView.js

```diff
@@ -42,6 +42,10 @@ foam.CLASS({
       name: 'sinksOnly',
       value: true
     },
+    {
```

Comment:
Good idea. See @abdelaziz-mahdy

Reply:
Pr to be merged after both changes are merged https://github.com/kgrgreer/foam3/pull/4261 uses it. <3 thanks kevin and ruby